### PR TITLE
Portfolio - Trinity Student Demo

### PR DESCRIPTION
**Pull request by trinity-edu-student-demo**

### CODE REVIEW

**Comment by vishal-reddy** [comment]
This code needs improvement. Please make changes in different areas that I have specified

File: week-4/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <head>
+        <title></title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <!-- minified CSS -->
+        <link rel="stylesheet" href="css/bootstrap.min.css">
+
+        <!-- Optional theme -->
+        <link rel="stylesheet" href="css/bootstrap-theme.min.css">
```
Comment (vishal-reddy) [inline week-4/index.html:12]: Please use the bootstrap-theme.css instead of the bootstrap-theme.min.css for dev purposes

Comment (trinity-edu-student-demo) [reply week-4/index.html:12]: But I want to include these changes for performance reasons please reconsider

Comment (vishal-reddy) [reply week-4/index.html:12]: I see what you mean. I think I am okay with this change

File: week-4/index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+       <!-- minified JavaScript -->
+        <script src="js/bootstrap.min.js"></script>
+    </head>
+    <body>
+        <h1>This Is My Portfolio</h1>
```
Comment (vishal-reddy) [inline week-4/index.html:18]: It is nice that you have a header. Please consider adding more design

**Comment by vishal-reddy** [comment]
Please make the changes that I have specified in the comments.

**Comment by trinity-edu-student-demo** [comment]
https://github.com/TrinityEdu/DigiNinja/pull/4

**Pull request closed without merging**